Zend/tests/isset_expr_error.phpt

```diff
@@ -5,4 +5,4 @@ Error message for isset(func())
 isset(1 + 1);
 ?>
 --EXPECTF--
-Fatal error: Cannot use isset() on the result of an expression (you can use "null !== expression" instead) in %s on line %d
+Fatal error: Cannot use isset() on the result of an expression (you can use "expression !== null" instead) in %s on line %d
```

Comment:
I got rid of the Yoda-style condition, because is not the primary style, as far as I noticed so far (thankfully :D)

Reply:
Not super fan of the usage of `you` from the original message. Maybe something like:
```suggestion
Fatal error: Cannot use isset() on the result of an expression ("expression !== null" can be used instead) in %s on line %d
```

Reply:
Thanks! This change makes sense for me!